timesketch/frontend-ng/src/components/Scenarios/Question.vue

```diff
@@ -15,67 +15,95 @@ limitations under the License.
-            <span style="font-size: 0.9em" v-bind="attrs" v-on="on">{{ question.display_name }}</span>
-          </template>
-          <v-card style="font-size: 0.9em" class="pa-3" width="400">
-            {{ question.description }}
-          </v-card>
-        </v-menu>
-      </div>
+    <div class="pa-2 pl-4" @click="expanded = !expanded" style="cursor: pointer">
+      <a style="font-size: 1em">{{ question.display_name }}</a>
+    </div>
 
-      <v-expand-transition>
-        <div v-show="expanded">
-          <div style="font-size: 0.9em" class="pa-4">
-            <v-textarea disabled outlined flat hide-details auto-grow rows="3">
-              <template v-slot:prepend-inner>
-                <v-avatar color="grey" class="mt-n1" size="28"></v-avatar>
-              </template>
-            </v-textarea>
-            <v-card-actions class="pl-0">
-              <v-btn disabled x-small outlined color="primary"> Yes </v-btn>
-              <v-btn disabled x-small outlined color="primary"> No </v-btn>
-              <v-btn disabled x-small outlined color="primary"> Inconclusive </v-btn>
-            </v-card-actions>
+    <v-expand-transition>
+      <div v-show="expanded">
+        <div class="ma-2 mx-4 mb-4 mt-n1">
+          <v-expand-transition>
+            <div v-if="fullDescription">
+              <small>{{ question.description }} <a @click="fullDescription = !fullDescription">show less</a></small>
+            </div>
+          </v-expand-transition>
+          <div v-if="!fullDescription">
+            <span>
+              <small>
+                {{ question.description.slice(0, 100) }}...
+                <a @click="fullDescription = !fullDescription">show more</a>
+              </small>
+            </span>
           </div>
         </div>
-      </v-expand-transition>
 
-      <v-divider></v-divider>
-    </div>
+        <v-card v-if="question.search_templates.length" flat outlined class="ma-2 mx-4 mb-6">
+          <v-system-bar dense flat> Get started </v-system-bar>
+          <div class="pa-3">
+            <ts-search-template
+              v-for="searchtemplate in question.search_templates"
+              :key="searchtemplate.id"
+              :searchtemplate="searchtemplate"
+            ></ts-search-template>
+          </div>
+        </v-card>
+
+        <!-- Commented out until we have conclusion API implemented
+        <div style="font-size: 0.9em" class="pa-4">
+          <v-textarea
+            outlined
+            flat
+            hide-details
+            auto-grow
+            rows="3"
+            placeholder="Add your conclusion"
+            style="font-size: 0.9em"
+          >
+            <template v-slot:prepend-inner>
+              <v-avatar color="grey" class="mt-n2 mr-2" size="28"></v-avatar>
+            </template>
+          </v-textarea>
+          <v-card-actions class="pl-0">
+            <v-btn x-small outlined color="primary"> Yes </v-btn>
```

Comment:
can we make that a CSS class maybe?

Reply:
css class? There is no extra styles on this element. x-small, outlined and color are all vuetify attributes.